pkgs/development/node-packages/default-v6.nix

```diff
@@ -44,6 +44,16 @@ nodePackages // {
     dontNpmInstall = true; # We face an error with underscore not found, but the package will work fine if we ignore this.
   });
 
+  yarn = nodePackages.yarn.overrideAttrs (oldAttrs: {
+    src = pkgs.fetchFromGitHub {
+      owner = "yarnpkg";
+      repo = "yarn";
+      rev = "v${oldAttrs.version}";
+      sha256 = "1h1qcgyx53hxkmg1z34k9b8gc3w0l670mb3jgmqlcbs6vjlqxbr8";
```

Comment:
How do you plan to keep this checksum up-to-date when `node-packages-v6.nix` is updated automatically?

Reply:
It would need to be updated like any other package, which is not ideal.